CMSgov/onemac
# Oy2 18484

Story: https://qmacbis.atlassian.net/browse/OY2-18484
Endpoint: https://d3teej317hrsw9.cloudfront.net/

### Details

Restore default table sorting by removing declaration of initialState from PortalTable component. This should be set as param during usage.

### Changes

- Removed initialState from PortalTable (this is passed as param by each usage of the component)
- Fixed PortalTable test which was wrong (presumably made to pass based on faulty initialState declaration)
- fixed id used for sort on package list

### Test Plan

1. Login as state submitter
2. Navigate to submission list and ensure table is default sorted on submitted date
